services/ui-src/src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+
+// components
+import {
+  Box,
+  Button,
+  Heading,
+  Modal as ChakraModal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import { CloseIcon } from "@cmsgov/design-system";
+import { makeMediaQueryClasses } from "utils";
+
+export const Modal = ({ actionFunction, content, modalState }: Props) => {
+  const mqClasses = makeMediaQueryClasses();
+  return (
+    <ChakraModal
+      data-testid="modal"
```

Comment:
Does it need a test id? Looks like we don't use it in the test

Reply:
Removed!